### ISSUE TITLE
 I'm curious how you get the rarity of each item

### ISSUE THREAD

**Issue by a361570441** [Neptunya/infinity-nikki-library#1]
Hello, I'm a developer and I'm curious how you get the rarity of each item and which set they belong to, I'd be grateful if you would tell me

**Comment by a361570441** [comment]
Did you get it by decrypting the return value of the /user/resource_entry request? If so, how did you do it?

**Comment by a361570441** [comment]
I tried to unpack the game, but I could only get the ID and picture of the item, but not the specific attributes of the item, such as rarity, source, etc.

**Comment by Neptunya** [comment]
I didn't unpack the game or anything of the sort. I just used [Python scripts](https://github.com/Neptunya/infinity-nikki-library/tree/main/python) to macro through menus and take screenshots to process the in-game text with OCR. I got the item name, rarity, and stats from the Glow Up menu. For rarity, I used cv2 to count the number of rarity stars. I got the image, label, and source from the compendium. As for outfits, I wrote that manually in a .json.
